src/EnergyPlus/DataEnvironment.hh

```diff
@@ -135,6 +135,7 @@ struct EnvironmentData : BaseGlobalStruct
     Real64 SkyTempKelvin = 0.0;                   // Current sky temperature {K}
     Real64 LiquidPrecipitation = 0.0;             // Current liquid precipitation amount (rain) {m}
     bool SunIsUp = false;                         // True when Sun is over horizon, False when not
+    bool SunIsUpPrevTS = false;                   // True when Sun is over horizon in the previous timestep, False when not in the previous timestep
```

Comment:
adding variable for sun status of the previous time step for use of checking "next morning"